relay/engines/docker.go

```diff
@@ -229,7 +229,31 @@ func (de *DockerEngine) attemptAuth() error {
 	return nil
 }
 
+func (de *DockerEngine) developerModeRefresh(bundle *config.Bundle) error {
+	if de.relayConfig.DevMode == true {
+		fullName := fmt.Sprintf("%s:%s", bundle.Docker.Image, bundle.Docker.Tag)
+		log.Warnf("Developer mode: Refreshing Docker image %s.", fullName)
+		err := de.pullImage(fullName)
+		if err != nil {
+			log.Errorf("Developer mode: Refresh of Docker image %s failed: %s.", fullName, err)
```

Comment:
Not a super huge deal, but maybe putting `"DEVELOPER MODE: "` in all caps would make those lines stand out a bit more in the logs.